### PR TITLE
Revisiting The Goliath: Or, that time I dripped out the SBC Starfury just because

### PR DESCRIPTION
**Pull request by unit0016**
## About The Pull Request
Drips the SHIT out of the SBC Starfury while not completely overhauling it. Touches everything ~~NOT in engineering or southward~~ (because I love how scuffed that part is and refuse to touch it on principle) - Also converts one map varedit into a real boy subtype, and moves tiny fans to their own file.

<details open>
<summary>Mandatory disclosure on the gameplay changes:</summary>

Fighters 1 and 3 are now NOT in the hangar, and are now attached to the formerly unused gunnery rooms.
Cryo now works. Yeah. I know.
You can actually open the anesthetic closet now.
Everyone now shares three spawners. This doesn't reduce the amount of people who can play when this rolls, as I've adjusted `var/uses` in accordance: it just reduces clutter.
A few of the horizontal double airlocks have been compacted into glass_large airlocks.
The bar windows now actually have grilles like they were meant to.
Four turbines have shown up. They aren't functional*, they just look like gunnery and conveniently fit in the spots. I'm sure this is space OSHA compliant.
The map is ever so slightly smaller, vertically. _This should distance us from an edge case where somehow all space levels are too cluttered for this to spawn properly, for the time being._

*Technically there's nothing stopping you from using them besides the amount of time it'd take for the operatives to kick your ass
</details>

<details open>
<summary>The Updated Map</summary>

</details>

Feel free to poke holes or request additional changes, this is primarily cosmetic.

## Why It's Good For The Game
This map was originally designed wayyy back before we even had the computer sprites we have now, (https://github.com/tgstation/tgstation/pull/27760 if you want to see SOUL) and it shows. While it will never have it's SM again, we can at least make the thing much nicer to look at.

## Changelog

:cl:
add: The SBC Starfury has been ever-so-slightly tweaked! Two of the shuttles are now where the gunnery bays used to be, and the rest of the differences are largely cosmetic. Largely.
code: Survival pod fans have been split off into their own file.
/:cl:

### CODE REVIEW

**Comment by ANAME593** [comment]
could this be a potential alternative to https://github.com/tgstation/tgstation/pull/68057

**Comment by unit0016** [comment]
linter fix time

**Comment by unit0016** [comment]
> could this be a potential alternative to #68057

nope, entirely unrelated

File: code/game/objects/structures/fans.dm [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+	buildstackamount = 2
+
+/obj/structure/fans/Initialize(mapload)
+	. = ..()
+	air_update_turf(TRUE, TRUE)
+
+/obj/structure/fans/Destroy()
+	air_update_turf(TRUE, FALSE)
+	. = ..()
+//Invisible, indestructible fans
```
Comment (san7890) [inline code/game/objects/structures/fans.dm:45]:
```suggestion
	. = ..()

//Invisible, indestructible fans
```
New line for clarity.

**Review by san7890 — CHANGES_REQUESTED** [review]
Hey there, looks very nice to me.

<details>
<summary>Click Me For Images</summary>

image

I really like the placement of this sign, nice work.

image

Please move this scrubber out from under the table.

image

Would rather you use the physical tinyfan structure that shows up in game.

image

Use a passive vent here.

image

This corner could use a small bit of light I feel, it's the main hanger and you don't want people tripping on potential slights.

image

Need lights here.

image

The docks for fighters One and Three end-caps have this really odd (unintentional?) look to them due to the smoothing, would you look into something that better fits this?

image

Would like if you could either move the vent or the mop+bucket.

image

Fighter 3 has these cameras that 1+2 do not have, is this intentional?

image

They also look _really_ bizarre in game.

image

Do they even have a defibrillator? I couldn't find one, not much sense having a mount for a non-existent bit of kit.

image

Yeah, this room is _really_ messy atmospherics wise, but could you try again to get the vent out from underneath the computer? Here's what I might try and do:

image

That may or may not work for you.

image

A small light bulb here (as well as on its symmetrical counterpart) would do a whole world of good.

image

The corvette seems to kinda... go _over_ the landing zone? Just feels a bit silly to me personally, is there a way to modify either the corvette or the bay to have it fit better? No worries if you can't figure it out, but it kinda hampers some aesthetics.

image

These tables blend, FYI. Might wanna make it all plastitanium?

image

Odd blending here if you can figure it out, no worries otherwise.

image

Plating here makes the pipes visible which feels silly, could you make this some sort of floor over? I think catwalk floors might look nice (might be a bit excessive though).

image

I'd honestly rather not have any pod windows, feels too excessive and cluttered. Your call though.

image

Doesn't really make sense for the button for these shutters to be in a completely separate room. Would be nice if you had a button for the "exterior" as well as the "interior" of that "SM" room.
</details>

I'm not sure if all of the external airlocks _should_ have tiny fans, since some are missing some (notably the ones that start with shuttles). Up to you.

Otherwise, solid bit of mapping. I'll be marking this PR as a Draft as a way to manage the PRs we have on our plate, feel free to mark it as "Ready for Review" again when this has been addressed.

**Comment by unit0016** [comment]
Gonna see what I can do here. The airlocks having tiny fans is gross and I wasn't aware of that, not my doing.

**Pull request merged**